include/api/CForecastRunner.h

```diff
@@ -74,28 +73,28 @@ class API_EXPORT CForecastRunner final : private core::CNonCopyable {
 public:
     //! max open forecast requests
     //! if you change this, also change the ERROR_TOO_MANY_JOBS message accordingly
-    static const size_t MAX_FORECAST_JOBS_IN_QUEUE = 3;
+    static const std::size_t MAX_FORECAST_JOBS_IN_QUEUE = 3;
 
     //! default expiry time
-    static const size_t DEFAULT_EXPIRY_TIME = 14 * core::constants::DAY;
+    static const std::size_t DEFAULT_EXPIRY_TIME = 14 * core::constants::DAY;
 
     //! max memory allowed to use for forecast models
-    static const size_t DEFAULT_MAX_FORECAST_MODEL_MEMORY = 20971520ull; // 20MB
+    static const std::size_t DEFAULT_MAX_FORECAST_MODEL_MEMORY;
```

Comment:
I think it is worth adding a comment that this isn't defined inline because we need its address.